iam_openapi_spec.yml

```diff
@@ -1693,6 +1693,8 @@ paths:
       operationId: getTokenForOrg
       parameters:
         - $ref: '#/components/parameters/organization_id'
+      requestBody:
+        $ref: '#/components/requestBodies/GetTokenForSubOrgRequest'
```

Comment:
OperationId and the request class name are contradicting. Simply name `GetTokenRequest`?